src/Me/components/Button/__snapshots__/Button.test.js.snap

```diff
@@ -12,6 +12,9 @@ exports[`Button component renders danger button when specified with skin prop 1`
   text-align: center;
   margin-bottom: 1rem;
   cursor: pointer;
+}
+
+.c1 {
```

Comment:
What is that? Does styledComponent changed the class name "hash" mechanizm?

Reply:
it does, yeah.  They are positional CSS selectors in this `c{number}` syntax.  Doing so prevents drift in the snapshots since styled-components does not guarantee consistent classname hashes.